NeedleInsertionRobot:
  ros__parameters:
    axis:
      x:
        speed:        2.5
        acceleration: 1.5
        deceleration: 1.5
        kP:            54
        kI:             4
        kD:           480
        limit:
          active: true
          min:    -100.0
          max:      30.0
      y:
        speed:        2.5
        acceleration: 1.5
        deceleration: 1.5
        kP:            15
        kI:             4
        kD:           332
        limit:
          active: true
          min:    -100.0
          max:      10.0
      z:
        speed:        2.5
        acceleration: 1.5
        deceleration: 1.5
        kP:            54
        kI:             4
        kD:           480
        limit:
          active: true
          min:    -0.05
          max:     0.05
      linear_stage:
        speed:        4.0
        acceleration: 0.2289
        deceleration: 0.2289
        kP:            54
        kI:             4
        kD:           480
        limit:
          active: true
          min:    -110.0
          max:       5.0

/stage:
  NeedleInsertionRobot:
    ros__parameters:
      axis:
        x:
          speed:        2.5
          acceleration: 1.5
          deceleration: 1.5
          kP:            54
          kI:             4
          kD:           480
          limit:
            active: true
            min:    -100.0
            max:      30.0
        y:
          speed:        2.5
          acceleration: 1.5
          deceleration: 1.5
          kP:            15
          kI:             4
          kD:           332
          limit:
            active: true
            min:    -100.0
            max:      10.0
        z:
          speed:        2.5
          acceleration: 1.5
          deceleration: 1.5
          kP:            54
          kI:             4
          kD:           480
          limit:
            active: true
            min:    -0.05
            max:     0.05
        linear_stage:
          speed:        4.0
          acceleration: 0.2289
          deceleration: 0.2289
          kP:            54
          kI:             4
          kD:           480
          limit:
            active: true
            min:    -110.0
            max:       5.0

/**:
  ros__parameters:
    axis:
      x:
        speed:        2.5
        acceleration: 1.5
        deceleration: 1.5
        kP:            54
        kI:             4
        kD:           480
        limit:
          active: true
          min:    -100.0
          max:      30.0
      y:
        speed:        2.5
        acceleration: 1.5
        deceleration: 1.5
        kP:            15
        kI:             4
        kD:           332
        limit:
          active: true
          min:    -100.0
          max:      10.0
      z:
        speed:        2.5
        acceleration: 1.5
        deceleration: 1.5
        kP:            54
        kI:             4
        kD:           480
        limit:
          active: true
          min:    -0.05
          max:     0.05
      linear_stage:
        speed:        4.0
        acceleration: 0.2289
        deceleration: 0.2289
        kP:            54
        kI:             4
        kD:           480
        limit:
          active: true
          min:    -110.0
          max:       5.0